control/trajectory_follower/lane_departure_checker/launch/lane_departure_checker.launch.xml

```diff
@@ -1,19 +1,17 @@
 <launch>
-  <arg name="input/twist" default="/localization/twist" />
+  <arg name="input/odometry" default="/localization/odometry" />
   <arg name="input/lanelet_map_bin" default="/map/vector_map" />
   <arg name="input/route" default="/planning/mission_planning/route" />
   <arg name="input/reference_trajectory" default="/planning/scenario_planning/trajectory" />
   <arg name="input/predicted_trajectory" default="/control/trajectory_follower/predicted_trajectory" />
-  <arg name="input/covariance" default="/localization/pose_with_covariance" />
   <arg name="config_file" default="$(find-pkg-share lane_departure_checker)/config/lane_departure_checker.param.yaml" />
 
   <node pkg="lane_departure_checker" exec="lane_departure_checker_node" name="lane_departure_checker_node" output="screen">
-    <remap from="~/input/twist" to="$(var input/twist)"/>
+    <remap from="~/input/odometry" to="$(var input/odometry)"/>
     <remap from="~/input/lanelet_map_bin" to="$(var input/lanelet_map_bin)"/>
     <remap from="~/input/route" to="$(var input/route)"/>
     <remap from="~/input/reference_trajectory" to="$(var input/reference_trajectory)"/>
     <remap from="~/input/predicted_trajectory" to="$(var input/predicted_trajectory)"/>
-    <remap from="~/input/covariance" to="$(var input/covariance)"/>
```

Comment:
@k0suke-murakami Could you check around `covariance` logic, please? :pray:

Reply:
Looks fine to me.  (From what I see from diff, there is no change in core covariance logic.)